Puppetfile

```diff
@@ -10,6 +10,10 @@ mod 'theforeman/dns',           :git => 'https://github.com/theforeman/puppet-dn
 mod 'theforeman/git',           :git => 'https://github.com/theforeman/puppet-git'
 mod 'theforeman/tftp',          :git => 'https://github.com/theforeman/puppet-tftp'
 
+# From git until PassengerMaxInstancesPerApp is available in released version
+#   https://github.com/puppetlabs/puppetlabs-apache/commit/d14e1a83e7e778fc6a000f8b28124fdb36834c43
+mod 'puppetlabs/apache',        :git => 'https://github.com/puppetlabs/puppetlabs-apache.git'
```

Comment:
If we really so want that one change (and I really would like to have serious prove it's worth it), we'd rather pull in that one change on top of the current 1.10.0 release into theforeman/puppetlabs-apache and use our fork in the installer.